### DISCUSSION TITLE
How to work with Spotify's API to display currently playing track on a website? | Leo's dev blog – stories, insights, and ideas

### DISCUSSION THREAD

**Discussion by giscus** [hta218/leohuynh.dev#78] [Announcements]
# How to work with Spotify's API to display currently playing track on a website? | Leo's dev blog – stories, insights, and ideas

Retrieve a Spotify access token to display the currently playing track on your website or application.

https://www.leohuynh.dev/snippets/using-spotify-api-to-display-currently-playing-track

**Comment by hta218** [comment]
#### Hey, thanks for checking out this post! 💡

If you found it useful (or have some ideas for improvement), I’d love to hear from you! Leave a comment below with your thoughts, or give a reaction to show your support. It means a lot to me!

Cheers 🍺
